docs/source/protocol.rst

```diff
@@ -316,3 +240,20 @@ Contents:
 - Singular Events (User ID, Event ID, Relative Timestamp, Auxiliary Payload)
 - Aggregate Events (User ID, Event ID, Count, Relative Timestamp, Auxiliary Payload)
 - Sequence Events (User ID, Event ID, Array of (Relative Timestamp, Auxiliary Payload))
+
+Version 3
+~~~~~~~~~
+
+- Endless X.X.X
```

Comment:
```suggestion
- Endless 4.0.0
```

Reply:
OK, I changed the `events.rst` page too.

Reply:
Seems to have been lost, still has x.x.x placeholder